build-docker.sh

```diff
@@ -18,8 +18,22 @@ if [ -z "$ALPINE_ARCH" ]; then
   esac
 fi
 
+if [ -z "$ALPINE_CHECKSUM" ]; then
+  case "$ALPINE_ARCH" in
+    aarch64)
+      ALPINE_CHECKSUM="0cc1dbad694c93b686bd76a0a5db3aaae2e1357e1b687a99b1bd37df9bae8692"
```

Comment:
Where did you get this hash from? I suggested a different one.

Reply:
I downloaded the wrong one, sorry for that. Yours was indeed the right one, somehow got confused by the file names. Should be fixed now.